### PR TITLE
tianmu compile: fix -Werror #101

### PR DESCRIPTION
**Pull request by DandreChen**
#101

### CODE REVIEW

**Comment by andyli029** [comment]
from 57

File: cmake/maintainer.cmake [hunk 1]
```diff
@@ -166,6 +166,8 @@ IF(MY_COMPILER_IS_CLANG)
   # -Wzero-as-null-pointer-constant
 ENDIF()
 
+SET(MY_WARNING_FLAGS "${MY_WARNING_FLAGS} -Wno-unknown-pragmas")
```
Comment (andyli029) [inline cmake/maintainer.cmake:169]: Add commets.

File: cmake/maintainer.cmake [hunk 2]
```diff
@@ -175,10 +177,10 @@ IF(MYSQL_MAINTAINER_MODE)
     STRING_APPEND(CMAKE_MODULE_LINKER_FLAGS " /WX")
     STRING_APPEND(CMAKE_SHARED_LINKER_FLAGS " /WX")
   ENDIF()
-  IF(MY_COMPILER_IS_GNU_OR_CLANG)
-    STRING_APPEND(MY_C_WARNING_FLAGS   " -Werror")
-    STRING_APPEND(MY_CXX_WARNING_FLAGS " -Werror")
-  ENDIF()
+#  IF(MY_COMPILER_IS_GNU_OR_CLANG)
```
Comment (andyli029) [inline cmake/maintainer.cmake:180]: Add comment TIANMU

Comment (DandreChen) [reply cmake/maintainer.cmake:182]: fix it

**Review by andyli029 — APPROVED** [review]
LGTM.

**Pull request merged**